src/main/java/com/rebecamontag/projectuniversity/entity/Registration.java

```diff
@@ -4,59 +4,45 @@
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
-
+import jakarta.persistence.SequenceGenerator;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.io.Serializable;
 import java.util.Date;
+import java.util.Objects;
 
 @Entity
-public class Registration {
```

Comment:
Esta entidade pode ser deletada, pois a relacao many to many ja esta mapeada em student